app/components/courses/search_result_component.html.erb

```diff
@@ -0,0 +1,49 @@
+<li class="app-search-results__item" data-qa="course">
+  <h2 class="app-search-result__item-title">
+    <%= govuk_link_to course_path(provider_code: course.provider_code, course_code: course.course_code), data: { qa: 'course__link' } do %>
+      <span class="app-search-result__provider-name" data-qa="course__provider_name"><%= helpers.smart_quotes(course.provider.provider_name) %></span>
+      <span class="app-search-result__course-name" data-qa="course__name"><%= course.decorate.display_title %></span>
+    <% end %>
+  </h2>
+  <dl class="app-description-list app-description-list--search-result">
+    <dt class="app-description-list__label">Study type</dt>
+    <dd data-qa="course__study_mode"><%= course.study_mode.humanize %></dd>
+    <dt class="app-description-list__label">Qualification</dt>
+    <dd data-qa="course__qualification">
+      <% if accredited_body %>
+        <p class="govuk-body"><%= course.qualification.humanize %></p>
+        <p class="govuk-hint"><%= accredited_body %></p>
+      <% else %>
+        <%= course.qualification.humanize %>
+      <% end %>
+    </dd>
+
+    <% if filtered_by_location? && has_sites? %>
+      <% if course.university_based? %>
+        <%= render partial: 'results/university', locals: { course: course } %>
+      <% else %>
+        <%= render partial: 'results/non_university', locals: { course: course } %>
+      <% end %>
+    <% end %>
+
+    <dt class="app-description-list__label">Financial support</dt>
+    <dd data-qa="course__funding_options"><%= course.decorate.funding_option %></dd>
+
+    <dt class="app-description-list__label">Degree required</dt>
+    <dd data-qa="course__degree_required"><%= degree_required %></dd>
+
+    <dt class="app-description-list__label">Visa sponsorship</dt>
+    <dd data-qa="course__visa_sponsorship"><%= visa_sponsorship %></dd>
+
+    <% if course['accrediting_provider'].present? %>
+      <dt class="app-description-list__label">Accredited body</dt>
+      <dd data-qa="course__accrediting_provider"><%= helpers.smart_quotes(course['accrediting_provider']['provider_name']) %></dd>
+    <% end %>
```

Comment:
This can be dropped, as this information is now [included above](https://github.com/DFE-Digital/find-teacher-training/pull/912/files#diff-0b2fa17210158aa5371db444bb8b86e846455fd18b05a4c940fe86a8d51f0174R15)